modules/admin/proc-rbac-ui-create-role.adoc

```diff
@@ -6,7 +6,7 @@ You can create a role in the {product} using the Web UI.
 .Prerequisites
 * You have an administrator role in the {product-short}.
 * You have installed the `@janus-idp/backstage-plugin-rbac` plugin in {product-short}. For more information, see link:{LinkPluginsGuide}[{NameOfPluginsGuide}].
-* You have configured the required permission policies. For more information, see xref:con-rbac-config-permission-policies_admin-rhdh[].
+* You have configured the required permission policies. For more information, see xref:con-rbac-config-permission-policies_{context}[].
```

Comment:
Will the RBAC content be placed under a different title?

Reply:
The RBAC content is not being relocated as part of this PR, but relocating the installation content did break an xref in the RBAC content. This fix enables a clean build. However, more extensive content and xref audits would be beneficial in the future.